src/TriggerActivityMakerChannelAdjacency.cpp

```diff
@@ -0,0 +1,241 @@
+/**
+ * @file TriggerActivityMakerChannelAdjacency.cpp
+ *
+ * This is part of the DUNE DAQ Application Framework, copyright 2021.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+
+#include "triggeralgs/ChannelAdjacency/TriggerActivityMakerChannelAdjacency.hpp"
+#include "TRACE/trace.h"
+#define TRACE_NAME "TriggerActivityMakerChannelAdjacencyPlugin"
+#include <vector>
+#include <math.h>
+
+using namespace triggeralgs;
+
+void
+TriggerActivityMakerChannelAdjacency::operator()(const TriggerPrimitive& input_tp,
+					     std::vector<TriggerActivity>& output_ta)
+{
+
+  // Add useful info about recived TPs here for FW and SW TPG guys.
+  if (m_print_tp_info){
+    TLOG(1) << " ########## m_current_window is reset ##########\n"
```

Comment:
```suggestion
    TLOG() << " ########## m_current_window is reset ##########\n"
```
Unless you want other level of logging, in which case use the new `TLOG_DEBUG(TLVL_` lines

Reply:
changed to `TLOG_DEBUG(TLVL_DEBUG_LOW)`